satpy/writers/ninjogeotiff.py

```diff
@@ -70,6 +69,16 @@
 Instead, pass those values in source units to the
 :func:`~satpy.enhancements.stretch` enhancement with the ``min_stretch``
 and ``max_stretch`` arguments.
+
+For images where the pixel value corresponds directly to a physical value,
+NinJo has a functionality to read the corresponding quantity (example:
+brightness temperature or reflectance).  To make this possible, the writer
+adds the tags ``Gradient`` and ``AxisIntercept``.  Those tags are added if
+and only if the image has mode ``L`` or ``LA`` and ``PhysicUnit`` is not set
+to ``"N/A"``.  In other words, to suppress those tags for images with mode
+``L`` or ``LA`` (for example, for the composite ``vis_with_ir``, where the
+physical interpretation of individual pixels is lost), one should set
+``PhysicUnit`` to ``"N/A"`` or ``"n/a"``.
```

Comment:
In cf conventions which we generally try to follow in satpy, no units is either "" or "1". Should we add these to n/a?

Reply:
Fixed in 0918261.